include/ui/veles_mainwindow.h

```diff
@@ -37,6 +41,20 @@ namespace ui {
 class MainWindowWithDetachableDockWidgets;
 
 /*****************************************************************************/
+/* QProxyStyleForDockWidgetWithIconOnTitleBar */
+/*****************************************************************************/
+
+class QProxyStyleForDockWidgetWithIconOnTitleBar : public QProxyStyle {
+  Q_OBJECT
+
+public:
+  QProxyStyleForDockWidgetWithIconOnTitleBar(QStyle* default_style);
+
+  void drawControl(QStyle::ControlElement element, const QStyleOption *option,
+      QPainter *painter, const QWidget *widget = Q_NULLPTR) const override;
```

Comment:
I think we use nullptr (as opposed to Q_NULLPTR) in other places (including in this exact file, line 113). I think it's better to be consistent (I do assume Q_NULLPTR expands to nullptr anyway).